plugins/GSdx/GSRenderer.cpp

```diff
@@ -274,7 +287,7 @@ bool GSRenderer::Merge(int field)
 
 		if(m_regs->SMODE2.INT && m_interlace > 0)
 		{
-			if (m_interlace == 7 && m_regs->SMODE2.FFMD == 1) // Auto interlace enabled / Odd frame interlace setting
+			if(m_interlace == 7 && m_regs->SMODE2.FFMD == 1) // Auto interlace enabled / Odd frame interlace setting
```

Comment:
> if it's not (a pointer for example) when not

Why isn't it desirable for pointers ? ``if(pointer)`` should be fine.

> Less code isn't always better. Especially if it still takes same amount of lines.

I don't think the ``==`` is actually better at this scenario. it's just _useless_.